empire.go

```diff
@@ -602,8 +642,12 @@ func (opts DeployOpts) Validate(e *Empire) error {
 
 // Deploy deploys an image and streams the output to w.
 func (e *Empire) Deploy(ctx context.Context, opts DeployOpts) (*Release, error) {
+	if err := authorizeApp(ctx, "Deploy", opts.App); err != nil {
```

Comment:
This would also currently allow you to create a new app by using `emp deploy` even if you didn't have the Create perm.